### PR TITLE
FAQ: add 4 questions and update description

### PR DESCRIPTION
**Pull request by lhyPingcap**
@shenli @lilin90  @lamxTyler PTAL

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -508,6 +508,46 @@ information_schema 库中的 tables 表里的 create_time 即为表的真实创
...
+1）通过在数据库中写 SQL 的方式来调整优先级：
+
+```
+select HIGH_PRIORITY | LOW_PRIORITY count(*) from table_name;
+insert HIGH_PRIORITY | LOW_PRIORITY into table_name insert_values;
+delete HIGH_PRIORITY | LOW_PRIORITY from table_name;
+update HIGH_PRIORITY | LOW_PRIORITY table_reference set assignment_list where where_condition;
+replace HIGH_PRIORITY | LOW_PRIORITY into table_name;
+```
+2）全表扫会自动调整为低优先级，analyze 也是默认低优先级。
```
Comment (lilin90) [inline FAQ.md]: Please leave a blank line between the code block and the "2）……".

File: FAQ.md [hunk 2]
```diff
@@ -508,6 +508,46 @@ information_schema 库中的 tables 表里的 create_time 即为表的真实创
...
+当表的 (修改数/当前总行数) 大于 `tidb_auto_analyze_ratio` 的时候，会自动触发 analyze 语句。`tidb_auto_analyze_ratio` 的默认值为 0，即关闭此功能。为了保险起见，在开启此功能的时候，保证了其最小值为 0.3。但是不能大于等于 `pseudo-estimate-ratio` (默认值为 0.7），否则会有一段时间使用 pseudo 统计信息，建议设置值为 0.5。
+
+#### 3.3.12 Select 语句在 TiDB 的 RC 与 RR 隔离级别下的实现过程，以及对应用的影响。
+
+TiDB 实现了两种隔离级别：读已提交和可重复读。
+
+Select 在不同的隔离级别上，会有不同的执行逻辑和表现形式：
+
+- 首先，select 在 RC 隔离级别下，不会到 PD 拿时间戳，直接用最大的 timestamp 去读数据，读取到当前最新的已经提交成功的数据。但在 RC 模式下，也会出现 RC 事务隔离级别本身的问题，即会出现幻读和不可重复读的情况。
+- RR 隔离级别还是要到 PD 去拿 timestamp ，通过 timestamp 以及 precolator 算法来读取多版本数据，防止读到被其他事务/会话修改过的数据，从而避免了幻读和不可重复读的问题。
```
Comment (lilin90) [inline FAQ.md]: - Delete the extra whitespace between "timestamp" and the following comma.
- Typo: precolator -> **Percolator**

File: FAQ.md [hunk 3]
```diff
@@ -508,6 +508,46 @@ information_schema 库中的 tables 表里的 create_time 即为表的真实创
...
+
+#### 3.3.11 在 TiDB 中 auto analyze 的触发策略是怎样的？
+
+触发策略：新表达到 1000 条，会自动触发。
+
+当表的 (修改数/当前总行数) 大于 `tidb_auto_analyze_ratio` 的时候，会自动触发 analyze 语句。`tidb_auto_analyze_ratio` 的默认值为 0，即关闭此功能。为了保险起见，在开启此功能的时候，保证了其最小值为 0.3。但是不能大于等于 `pseudo-estimate-ratio` (默认值为 0.7），否则会有一段时间使用 pseudo 统计信息，建议设置值为 0.5。
+
+#### 3.3.12 Select 语句在 TiDB 的 RC 与 RR 隔离级别下的实现过程，以及对应用的影响。
+
+TiDB 实现了两种隔离级别：读已提交和可重复读。
```
Comment (lilin90) [inline FAQ.md]: Add English expressions to make them more friendly to readers:

读已提交和可重复读 -> **读已提交 (Read Committed, RC) 和可重复读 (Repeatable Read, RR)**

File: FAQ.md [hunk 4]
```diff
@@ -508,6 +508,46 @@ information_schema 库中的 tables 表里的 create_time 即为表的真实创
...
+
+Select 在不同的隔离级别上，会有不同的执行逻辑和表现形式：
+
+- 首先，select 在 RC 隔离级别下，不会到 PD 拿时间戳，直接用最大的 timestamp 去读数据，读取到当前最新的已经提交成功的数据。但在 RC 模式下，也会出现 RC 事务隔离级别本身的问题，即会出现幻读和不可重复读的情况。
```
Comment (lilin90) [inline FAQ.md]: 直接 -> **而是**直接

File: FAQ.md [hunk 5]
```diff
@@ -508,6 +508,46 @@ information_schema 库中的 tables 表里的 create_time 即为表的真实创
...
+update HIGH_PRIORITY | LOW_PRIORITY table_reference set assignment_list where where_condition;
+replace HIGH_PRIORITY | LOW_PRIORITY into table_name;
+```
+2）全表扫会自动调整为低优先级，analyze 也是默认低优先级。
+
+#### 3.3.11 在 TiDB 中 auto analyze 的触发策略是怎样的？
+
+触发策略：新表达到 1000 条，会自动触发。
+
+当表的 (修改数/当前总行数) 大于 `tidb_auto_analyze_ratio` 的时候，会自动触发 analyze 语句。`tidb_auto_analyze_ratio` 的默认值为 0，即关闭此功能。为了保险起见，在开启此功能的时候，保证了其最小值为 0.3。但是不能大于等于 `pseudo-estimate-ratio` (默认值为 0.7），否则会有一段时间使用 pseudo 统计信息，建议设置值为 0.5。
```
Comment (lilin90) [inline FAQ.md]: For the parentheses in this line, modify them to Chinese parentheses, and don't leave extra whitespaces before or after them.

File: FAQ.md [hunk 6]
```diff
@@ -508,6 +508,46 @@ information_schema 库中的 tables 表里的 create_time 即为表的真实创
...
 TiDB 在执行 SQL 时，预估出来每个 operator 处理了超过 10000 条数据就认为这条 query 是 expensive query。可以通过修改 tidb-server 配置参数来对这个门限值进行调整，调整后需要重新启动 tidb-server。
 
+#### 3.3.10 在 TiDB 中如何控制或改变 SQL 提交的执行优先级？
+
+首先需要了解高优先级和低优先级的语法元素
+
+- HIGH_PRIORITY：该语句为高优先级语句，TiDB 在执行阶段会优先处理这条语句
+- LOW_PRIORITY：该语句为低优先级语句，TiDB 在执行阶段会降低这条语句的优先级
+
+以上两种参数可以结合 TiDB 的 DML 语言进行使用，具体使用方式可以参考[官方文档](https://pingcap.com/docs-cn/sql/dml/)，使用方法举例如下：
```
Comment (lilin90) [inline FAQ.md]: Use relative path for the link: `[官方文档](sql/dml.md)`

File: FAQ.md [hunk 7]
```diff
@@ -508,6 +508,46 @@ information_schema 库中的 tables 表里的 create_time 即为表的真实创
...
+```
+2）全表扫会自动调整为低优先级，analyze 也是默认低优先级。
+
+#### 3.3.11 在 TiDB 中 auto analyze 的触发策略是怎样的？
+
+触发策略：新表达到 1000 条，会自动触发。
+
+当表的 (修改数/当前总行数) 大于 `tidb_auto_analyze_ratio` 的时候，会自动触发 analyze 语句。`tidb_auto_analyze_ratio` 的默认值为 0，即关闭此功能。为了保险起见，在开启此功能的时候，保证了其最小值为 0.3。但是不能大于等于 `pseudo-estimate-ratio` (默认值为 0.7），否则会有一段时间使用 pseudo 统计信息，建议设置值为 0.5。
+
+#### 3.3.12 Select 语句在 TiDB 的 RC 与 RR 隔离级别下的实现过程，以及对应用的影响。
```
Comment (lilin90) [inline FAQ.md]: Make it sounds like a question:

影响。 -> **影响是怎样的？**

File: FAQ.md [hunk 8]
```diff
@@ -508,6 +508,47 @@ information_schema 库中的 tables 表里的 create_time 即为表的真实创
 
 TiDB 在执行 SQL 时，预估出来每个 operator 处理了超过 10000 条数据就认为这条 query 是 expensive query。可以通过修改 tidb-server 配置参数来对这个门限值进行调整，调整后需要重新启动 tidb-server。
 
+#### 3.3.10 在 TiDB 中如何控制或改变 SQL 提交的执行优先级？
+
+首先需要了解高优先级和低优先级的语法元素
```
Comment (lilin90) [inline FAQ.md]: It would be better to add a colon "：" after "元素".

File: FAQ.md [hunk 9]
```diff
@@ -508,6 +508,47 @@ information_schema 库中的 tables 表里的 create_time 即为表的真实创
...
+
+#### 3.3.11 在 TiDB 中 auto analyze 的触发策略是怎样的？
+
+触发策略：新表达到 1000 条，会自动触发。
+
+当表的（修改数/当前总行数）大于 `tidb_auto_analyze_ratio` 的时候，会自动触发 analyze 语句。`tidb_auto_analyze_ratio` 的默认值为 0，即关闭此功能。为了保险起见，在开启此功能的时候，保证了其最小值为 0.3。但是不能大于等于 `pseudo-estimate-ratio`（默认值为 0.7），否则会有一段时间使用 pseudo 统计信息，建议设置值为 0.5。
+
+#### 3.3.12 Select 语句在 TiDB 的 RC 与 RR 隔离级别下的实现过程，以及对应用的影响是怎样的？
+
+TiDB 实现了两种隔离级别：读已提交（Read Committed, RC）和可重复读（Repeatable Read, RR）。
```
Comment (lilin90) [inline FAQ.md]: If the content inside the parentheses is all in English, use the English parentheses and leave a whitespace outside the parentheses.

File: FAQ.md [hunk 10]
```diff
@@ -508,6 +508,47 @@ information_schema 库中的 tables 表里的 create_time 即为表的真实创
...
+
+2）全表扫会自动调整为低优先级，analyze 也是默认低优先级。
+
+#### 3.3.11 在 TiDB 中 auto analyze 的触发策略是怎样的？
+
+触发策略：新表达到 1000 条，会自动触发。
+
+当表的（修改数/当前总行数）大于 `tidb_auto_analyze_ratio` 的时候，会自动触发 analyze 语句。`tidb_auto_analyze_ratio` 的默认值为 0，即关闭此功能。为了保险起见，在开启此功能的时候，保证了其最小值为 0.3。但是不能大于等于 `pseudo-estimate-ratio`（默认值为 0.7），否则会有一段时间使用 pseudo 统计信息，建议设置值为 0.5。
+
+#### 3.3.12 `SELECT` 语句在 TiDB 的 RC 与 RR 隔离级别下的实现过程，以及对应用的影响是怎样的？
```
Comment (shenli) [inline FAQ.md]: RC 在 2.0 中已经禁用了

File: FAQ.md [hunk 11]
```diff
@@ -508,6 +508,47 @@ information_schema 库中的 tables 表里的 create_time 即为表的真实创
...
+delete HIGH_PRIORITY | LOW_PRIORITY from table_name;
+update HIGH_PRIORITY | LOW_PRIORITY table_reference set assignment_list where where_condition;
+replace HIGH_PRIORITY | LOW_PRIORITY into table_name;
+```
+
+2）全表扫会自动调整为低优先级，analyze 也是默认低优先级。
+
+#### 3.3.11 在 TiDB 中 auto analyze 的触发策略是怎样的？
+
+触发策略：新表达到 1000 条，会自动触发。
```
Comment (shenli) [inline FAQ.md]: 我记得还有要求多久之内没有写入，可以找海滨确认一下

File: FAQ.md (hunk 10 above)
Comment (lhyPingcap) [reply FAQ.md]: 那这个问题暂时先不放官网 FAQ 了

File: FAQ.md (hunk 11 above)
Comment (lamxTyler) [reply FAQ.md]: 需要在 20*stats-lease 的时间内没有写入，也就是默认 1min.

**Pull request merged**